index.html

```diff
@@ -0,0 +1,65 @@
+        dog that is always willing to play and spend time with her family.</p>
+
+        <div class="container">
+        <img src="imagenes/BetinaPuppy1.jpg" alt="Puppy Betina-1" class="img_pup">
+        <img src="imagenes/BetinaPuppy2.jpg" alt="Puppy Betina-2" class="img_pup">
+        </div>
+
+        <p>Betina arrived at her home at 3 months of age, since then she has become a very beloved member of the family. Despite her small size,
+        Betina has a great personality and is very funny, she is able to make anyone laugh with her mischief and games.
+        Since she arrived at her home, Betina has been a very loving and loyal dog, she is always willing to give and receive love, she is a very beloved dog by all members of the family
+        and by friends who know her. Betina is a very fun and playful dog, she is always willing to play and have a good time with her family, she is especially loving with children
+        and gets along very well with other animals. Betina is a very special dog for her family, since she arrived she has been a great contribution to her home and a great companion for everyone.</p>
+
+        <h2>Some of Betina's favorite things</h2>
+
+        <ul>
+        <li><strong>Fetching the ball:</strong> Betina is very active and loves playing fetch, it's one of her favorite pastimes.</li>
+        <li><strong>Named toys:</strong> Betina has a wide variety of toys that she recognizes by name, such as "fox", "paws", "mole" and "bear", she loves playing with them and they are an important part of her life.</li>
+        <li><strong>Learning new tricks and commands:</strong> Betina is very intelligent and loves learning new tricks and commands, she is a very smart dog.</li>
+        <li><strong>Spending time with her family:</strong> Betina is very loving and enjoys spending time with her family, especially with her owner, she is a very loyal and faithful dog.</li>
+        </ul>
+
+        <h2>Things that Betina dislikes</h2>
+    
+        <ol>
+        <li><strong>noises:</strong> Betina may be scared of loud noises such as thunder or fireworks.</li>
+        <li><strong>Being alone for long periods of time:</strong> Betina likes the company and may suffer from separation anxiety if left alone for too long.</li>
+        <li><strong>Being dirty:</strong> Betina may not like getting dirty and may not enjoy playing in mud or dirty environments.</li>
+        <li><strong>Strict discipline:</strong> Betina may not respond well to strict discipline or harsh training methods. She may be better trained with positive reinforcement and rewards-based methods.</li>
+        </ol>
+
+        <hr>
+
+        <div>
+        <button class="btn">Click if you liked<img src="imagenes/thumbs-up-solid.svg" alt="ThumbsUp" class="thumbs"></button>
+        </div>
+
+
+
+</main>
+
+</body>
```

Comment:
Kindly remove the empty lines in your HTML code. Even though the empty lines make it easier for you to see the entire structure, it is best practice to remove the empty lines once you're done with that section of the project.